pkgs/test/pubspec.yaml

```diff
@@ -8,7 +8,7 @@ environment:
   sdk: '>=2.2.0 <3.0.0'
 
 dependencies:
-  analyzer: ">=0.26.4 <0.37.0"
+  analyzer: ^0.37.0
```

Comment:
We should just expand the upper bound if possible, otherwise people will be blocked on upgrading until all transitive deps are updated.

Reply:
Done.